public/css/gogs.css

```diff
@@ -1311,6 +1311,8 @@ footer .ui.language .menu {
 .repository.file.list #file-content .code-view .lines-num .hljs li,
 .repository.file.list #file-content .code-view .lines-code .hljs li {
   padding-left: 5px;
+  display: inline-block;
```

Comment:
Apologies, I misunderstood your comment. I've moved the changes to the `.less` file.